src/Ombi.Helpers/StringHelper.cs

```diff
@@ -77,6 +77,57 @@ public static int IntParseLinq(string stringIn)
             return -1;
         }
 
+        public static string BuildEpisodeList(IEnumerable<(int EpisodeNumber, DateTime? Aired)> orderedEpisodes)
```

Comment:
Looks good, but don't feel like this belongs in the `StringHelper`? It's very specific to what's going on